caps/templates/caps/council_detail.html

```diff
@@ -140,14 +143,17 @@ <h2 class="mr-auto pr-3 mb-1 mb-sm-0">Climate documents</h2>
                                       <span class="mx-2">&middot;</span>
                                       {{ plandocument.get_scope|title }}
                                     {% endif %}
-                                    {% if plandocument.is_deleted %}
+                                    {% if plandocument.is_deleted and not plandocument.file_exists %}
                                       <span class="mx-2">&middot;</span>
                                       We no longer hold a copy of this document
                                     {% endif %}
                                 </p>
                             </div>
 
                       {% if plandocument.is_deleted %}
+                        {% if not plandocument.file_exists %}
```

Comment:
@struan I _think_ this is meant to be `{% if plandocument.file_exists %}`?

Reply:
doh! Fixed